src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -757,15 +756,19 @@ internal override MemberSemanticModel GetMemberModel(CSharpSyntaxNode node)
                     case SyntaxKind.ConstructorDeclaration:
                         {
                             ConstructorDeclarationSyntax constructorDecl = (ConstructorDeclarationSyntax)memberDecl;
+                            var expressionBody = constructorDecl.GetExpressionBodySyntax();
                             return GetOrAddModelForParameterDefaultValue(constructorDecl.ParameterList.Parameters, span) ??
-                                GetOrAddModelIfContains(constructorDecl.Initializer, span) ??
-                                GetOrAddModelIfContains(constructorDecl.Body, span);
+                                   GetOrAddModelIfContains(constructorDecl.Initializer, span) ??
```

Comment:
@agocke I think they are all a bit different. Constructors have initializers, methods have parameters and destructors don't have parameters - at least in the former implementation.